Description: relax test_ln
 test_ln fails on i386, presumablly due to x87's unusual and not entirely
 deterministic rounding behaviour

 We widen the range used in the test by one unit in last place.
Author: Peter Michael Green <plugwash@debian.org>

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1940,3 +1940,4 @@
                 // for this branch, -pi <= arg(ln(z)) <= pi
-                assert!(-f64::consts::PI <= c.ln().arg() && c.ln().arg() <= f64::consts::PI);
+                let piplusepsilon = f64::from_bits(f64::consts::PI.to_bits()+1);
+                assert!(-piplusepsilon <= c.ln().arg() && c.ln().arg() <= piplusepsilon);
             }
